d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
+					font-family: inherit;
+					font-size: 0.8rem;
+					font-weight: 400;
+					line-height: 1rem;
```

Comment:
I'm not seeing any cut off on all supported browsers and IE 11 (with line-height: 1rem and testing both regular compact font and compact with d2l-dyslexic class). Are there any specific steps to potentially reproduce that I could have missed?